etc/module.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+    <module name="Graycore_GuestOrdersGraphQl" setup_version="0.0.1">
+        <sequence>
+            <module name="Magento_Framework"/>
+            <module name="Magento_GraphQl"/>
+            <module name="Magento_Quote"/>
+            <module name="Magento_SalesGraphQl"/>
```

Comment:
@damienwebdev I don't need to explicitly reference `Magento_Sales`, right? I assume that it will be pulled in by `Magento_SalesGraphQl`.

Reply:
Correct.